assets/app/scripts/services/data.js

```diff
@@ -150,6 +150,9 @@ angular.module('openshiftConsole')
     this._getNamespace(resource, context, opts).then(function(ns){
       $http(angular.extend({
```

Comment:
Should be fixed now by following you suggestion below

https://github.com/openshift/origin/pull/4999#discussion-diff-41781690